mainpage.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+  <meta charset="utf-8">
+  <title>Best Recipes</title>
+  <link rel = "stylesheet" href = "stylesheet.css">
+</head>
+<!-- Creates a background for the entire page, reused for the other pages -->
+<body class = "background">
+  <!-- Creates a box at the top for header and nav -->
+  <div class = "container">
+
+    <header>
+
+      <h2 class = "flex-container">
+
+        Shafay's Chowdown
+
+      </h2>
+
+
+      <nav>
+
+        <a class = "flex-container2" href = "mainpage.html">Home</a>
+        <a class = "flex-container2" href = "about.html">About Me</a>
+  
+      </nav>
+
+
+    </header>
+
+  </div>
```

Comment:
This navbar could use a bit of cleaning up. Right now there are a couple redundant containers (you don't need the div wrapped around the header) and some of the CSS rules you wrote don't do anything. Structurally, I would reccomend something like this:
```
<nav>
  <h2>Shafay's Chowdown</h2>
  <div>
    <a>Home</a>
    <a>About Me </a>
  </div>
</nav>
```
You don't need to resubmit milestone 1, but please get this fixed for milestone 3 and reach out if you have any questions.